File: python/String/10809.py
import sys


if __name__ == "__main__":
    num=input()
    a=[-1 for  i in range(26)]
    
    index=0
    for val in num:
      if (a[ord(val)-97]==-1):  # a == Ascii (97)
        a[ord(val)-97]=index

      index+=1

    for val in a:
      print(val,end=" ")

File: Go/String/10809.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	var list []int
	for i := 0; i < 26; i++ {
		list = append(list, -1)
	}
	var input string
	reader := bufio.NewReader(os.Stdin)
	fmt.Fscan(reader, &input)

	for index, val := range input {
		if list[int(val)-97] == -1 {
			list[int(val)-97] = index
		}
	}

	for _, val := range list {
		fmt.Printf("%d ", val)
	}

}
